### PR TITLE
Update for IPSOS tags

### PR DESCRIPTION
**Pull request by amathot**
Updated the script so it would not pseudotranslate IPSOS specific markup such as {setTheTrap}, {CATEGORY}, {QUESTION_NUMBER}...

### CODE REVIEW

File: pseudoxlate/pseudoxlate.groovy [hunk 1]
```diff
@@ -18,17 +18,17 @@ def gui(){
...
 
 		// pseudo-translate lower case letters 
-		search = /\p{Ll}(?![^<]*?>|[^%\s]*?%)/   
+		search = /\p{Ll}(?![^<]*?>|[^%\s]*?%|[^\{]*?\})/   
 		replac = "x"
 		target = target.replaceAll(search, replac)
 
 		// pseudo-translate ditis (in any case ;)
-		search = /\p{N}(?![^<]*?>|[^%\s]*?%)/   
+		search = /\p{N}(?![^<]*?>|[^%\s]*?%|[^\{]*?\})/
```
Comment (msoutopico) [inline pseudoxlate/pseudoxlate.groovy]: Good change. This is fine, except for the backslash inside the class, you don't need to escape curly braces inside a class. You can simply have  `[^{]*?\}`.

File: pseudoxlate/pseudoxlate.groovy [hunk 2]
```diff
@@ -68,5 +68,5 @@ def gui() {
 
 // Check
 // [a-wyzA-WYZ0-9]
-// [!-’,;.:“”\(\)\]\[\?&=xX#–"º%…—/   ^t^13  ]
+// [!-’,;.:“�\(\)\]\[\?&=xX#–"º%…—/   ^t^13  ]
```
Comment (msoutopico) [inline pseudoxlate/pseudoxlate.groovy]: These changes seem to be unintended, caused by some encoding issue when you paste copied text. Not approved.

**Review by msoutopico — CHANGES_REQUESTED** [review]
Thank you for your addition, Adri. Good one! Before accepting the changes, please update the regex (just remove the first backslash) and please remove the last two changes (due to some encoding issue).

**Review by amathot — COMMENTED** [review]
Changes made :)

**Review by msoutopico — APPROVED** [review]
Looks good now. Thanks!

**Pull request merged**